pages/ksphere/kommander/1.1/clusters/attach-cluster/federated-addons/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: Federated Addons
+title: Federated Addons
+menuWeight: 7
```

Comment:
I don't know what this means so I'm just leaving a comment here to double-check.

Reply:
I did and it makes little sense but `7` looks OK:
```
➜  1.1 (armandgrillet/D2IQ-70223) ✔ pwd
/Users/armand/Code/dcos-docs-site/pages/ksphere/kommander/1.1
➜  1.1 (armandgrillet/D2IQ-70223) ✔ pt "menuWeight: 7"
./centralized-cost-monitoring/index.md
5:menuWeight: 7

./centralized-monitoring/index.md
5:menuWeight: 7

./clusters/creating-konvoy-cluster-aws/index.md
5:menuWeight: 7

./clusters/creating-konvoy-cluster-on-prem/index.md
5:menuWeight: 7

./clusters/creating-konvoy-cluster-azure/index.md
5:menuWeight: 7

./clusters/attach-eks-cluster/index.md
5:menuWeight: 7

./clusters/index.md
5:menuWeight: 7

./clusters/attach-cluster/index.md
5:menuWeight: 7

./clusters/attach-cluster/federated-addons/index.md
5:menuWeight: 7

➜  1.1 (armandgrillet/D2IQ-70223) ✔ pt "menuWeight: 8"
./projects/index.md
5:menuWeight: 8

./workspaces/index.md
5:menuWeight: 8

➜  1.1 (armandgrillet/D2IQ-70223) ✔ pt "menuWeight: 6"
./licensing/index.md
6:menuWeight: 6
```

Reply:
@russurquhart1 Do you have any information regarding the use of `menuWeight`? Would like to confirm which value to use in this case.

Reply:
> @russurquhart1 Do you have any information regarding the use of `menuWeight`? Would like to confirm which value to use in this case.

@armandgrillet menuweight determines the order of the TOC items, by level. So you should never have 2 items with the same menuweight. Each level starts over:

Release Notes (5) 
Introduction (10)
Installing (15)
  Install on AWS (5)
  Install on Azure (10) 
  Install on-prem (15) 
Troubleshooting (20)

Reply:
@cmastr Thanks for the info! Updating this PR one more time to update this and move "Attach Amazon EKS Cluster to Kommander"